server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -178,11 +193,15 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
         if (this.serviceConfiguration.deli.opEvent.enable) {
             this.updateOpMaxTimeTimer();
         }
+
+        this.isNewDocument = this.sequenceNumber === 0;
+
+        this.logSessionStartMetrics();
```

Comment:
Could you call this around an `if (this.serviceConfiguration.enableLambdaMetrics) { }` statement. Otherwise it will create `sessionStartMetric`